addons/iot_drivers/main.py

```diff
@@ -74,6 +74,9 @@ def _get_changes_to_send(self):
     def send_all_devices(self, server_url=None):
```

Comment:
This should be converted to a private method:
- It doesn't look use in the webclient, nor js nor xml nor html
- It allows users/attackers to choose arbitrary the `server_url`.